### ISSUE TITLE
error: cannot find -lagora_rtc_sdk

### ISSUE THREAD

**Issue by rhenrhee** [AgoraIO/Basic-Video-Broadcasting#132]
Windows 7 Ultimate, Qt5.15
In the process of building the project errors appear: 
error: cannot find -lagora_rtc_sdk
:-1: error: collect2.exe: error: ld returned 1 exit status
:-1: error: [Makefile.Debug:109: debug/OpenLive.exe] Error 1

**Comment by yoreland** [comment]
Just for double-confirm, do you Unzip the downloaded SDK package and copy the libs to the project folder?

**Comment by rhenrhee** [comment]
> Just for double-confirm, do you Unzip the downloaded SDK package and copy the libs to the project folder?

Yes, of course. To make sure everything was done according to the instructions, I additionally ran the related project under "MFC" and it ran correctly. I did the same experiment for the Qt project under Microsoft Windows 10 and the error is the same. Perhaps I should give more detailed instructions or record a video. At the same time, for Mac OS X all versions work correctly!

**Comment by sbd021** [comment]
SDK version? Which branch?

**Comment by sbd021** [comment]
You need to copy libs folder in sdk zip to the path OpenLive.pro is located in. The compiler will find agora_rtc_sdk.lib from libs/x86(x64_86).

**Comment by rhenrhee** [comment]
Thank you for your reply. I'll check today and report back.
